nepalingo-web/src/components/FeedbackForm.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+
+
+interface FeedbackFormProps {
+  onClose: () => void;
+}
+
+
+const FeedbackForm: React.FC<FeedbackFormProps> = ({ onClose }) => {
+  const [rating, setRating] = useState<number | null>(null);
+  const [emojiRating, setEmojiRating] = useState<number | null>(null);
+  const [comments, setComments] = useState<string>('');
+  const [submitted, setSubmitted] = useState<boolean>(false);
+
+
+  const handleStarRating = (rate: number) => {
+    setRating(rate);
+  };
+
+
+  const handleEmojiRating = (rate: number) => {
+    setEmojiRating(rate);
+  };
+
+
+  const handleSubmit = () => {
+
+    setSubmitted(true);
+    setTimeout(onClose, 5000);
+  };
+
+
+  return (
+    <div className="bg-white p-6 rounded-lg">
+      {submitted ? (
+        <div className="text-center">
+          <p className="text-lg font-semibold mb-4">Thank you for your feedback! 🎉 </p>
```

Comment:
The closing after 5 seconds is too long and kindof unexpected behaviour haha. Maybe we should have a close button on this div. Like an x on the top right or something

Reply:
Done Thankyou!